### PR TITLE
Convert XML element values to strs when composing input data

### PR DESCRIPTION
**Pull request by ekoval**
Plus some tests cleanup

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: teamsupport/config.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-ORG_ID = None
-AUTH_KEY = None
+ORG_ID = 1114960
+AUTH_KEY = '2da0d579-1507-4d81-9112-a3e47f62d636'
```
Comment (highflyingaction) [inline teamsupport/config.py]: This is public repo so we should probably get a new auth key from team support since this one has been exposed.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: teamsupport/utils.py [hunk 2]
```diff
@@ -7,6 +7,6 @@ def to_xml(root, data):
 
     for k, v in iteritems(data):
         child = etree.SubElement(xml, k)
-        child.text = v
+        child.text = str(v)
```
Comment (highflyingaction) [inline teamsupport/utils.py]: > `text`: Text before the first subelement. This is either a string or the value None, if there was no text.

from http://lxml.de/api/lxml.etree._Element-class.html

This should probably check for `None` before converting to a string.

Comment (ekoval) [reply teamsupport/utils.py]: and what if it `is None`?

Comment (ekoval) [reply teamsupport/utils.py]: `text = ''`?

Comment (highflyingaction) [reply teamsupport/utils.py]: I would expect that `child.text = None` and `child.text = ''` would generate equivalent XML.  Try it and find out.

Comment (stefanor) [reply teamsupport/utils.py]: This is python 2, shouldn't this be unicode?

Comment (highflyingaction) [reply teamsupport/utils.py]: Its 2 and 3 actually so `six.text_type`.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: teamsupport/utils.py [hunk 3]
```diff
@@ -7,6 +7,6 @@ def to_xml(root, data):
 
     for k, v in iteritems(data):
         child = etree.SubElement(xml, k)
-        child.text = v
+        child.text = str(v) if v is not None else v
```
Comment (highflyingaction) [inline teamsupport/utils.py]: I think it is worth adding a test for when `v` is `None`.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Review by highflyingaction — APPROVED** [review]
